cypress.json

```diff
@@ -3,7 +3,7 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "video": false,
-  "defaultCommandTimeout": 81000,
+  "defaultCommandTimeout": 101000,
```

Comment:
+20 sec for waiting makes our test slower(

Reply:
yes but without this waiting data import tests are failed